class.jetpack.php

```diff
@@ -3790,7 +3793,7 @@ function upload_handler( $update_media_item = false ) {
 
 		$media_keys = array_keys( $_FILES['media'] );
 
-		$token = Jetpack_Data::get_access_token( get_current_user_id() );
+		$token = self::init()->connection_manager->get_access_token( get_current_user_id() );
```

Comment:
This feels pretty noisy, both in terms of the `self::init()` part, and the length of the connection_manager prop, and the fact that the prop is accessed as a var rather than through a method.

What about something like:

```php
self::connection()->get_access_token( get_current_user_id() );
```

and elsewhere:

```
static function connection() {
  // maybe memoize this?
  self::init();
  return new Connection_Manager();
}
```

Reply:
Then anywhere else in the code we like, we can call Jetpack::connection() to get a connection manager instance, at least as shorthand unless the alternative is necessary.

Reply:
Alternatively, a static entry point for connections:

```php 
// in a package file, e.g. Automattic/Jetpack/Connection.php
namespace Automattic\Jetpack;
use \Automattic\Jetpack\Connection\Manager as Connection_Manager;

function Connection() {
  return new Connection_Manager();
}

class Connection {
// dummy class?
}
```

```php
// this acts as a handy singleton in other places, and doesn't need a constructor, but 
// unfortunately can't be aliased or imported, so requires a dummy class in the same file
$token = Connection()->get_access_token( $user_id );
```